run_brer/run_config.py

```diff
@@ -377,18 +377,25 @@ def __train(self, tpr_file=None, **kwargs):
             # of alpha.
             # noinspection PyUnresolvedReferences
             current_alpha = context.potentials[i].alpha
+            if current_alpha == 0.0:
+                raise RuntimeError('Alpha value was constrained to 0.0, which indicates something went wrong')
+
             # noinspection PyUnresolvedReferences
             current_target = context.potentials[i].target
+            current_converged = getattr(context.potentials[i], 'converged', None)
 
             self.run_data.set(name=current_name, alpha=current_alpha)
             self.run_data.set(name=current_name, target=current_target)
+            self.run_data.set(name=current_converged, converged=current_converged)
```

Comment:
Recall that we only `RunData.set()` parameters that are described in that class and which are read and written in the state file. We can't initialize the `converged` property of the plugin potential, so we haven't bothered to add it to the RunData parameters.
```suggestion
```